x-pack/plugins/ml/public/application/components/anomalies_table/links_menu.tsx

```diff
@@ -636,6 +667,28 @@ export const LinksMenuUI = (props: LinksMenuProps) => {
       }
     }
 
+    if (props.sourceIndicesWithGeoFields[anomaly.jobId]) {
```

Comment:
For `lat_long` anomalies, where we already have the `View in Maps` action to open a map using the geo field that was used in the job, I wonder if it is a bit confusing to show this action too? Admittedly it can show different information if there is more than one geo point field in the data view, but I lean towards not showing this new action if we already have the `View in Maps` item e.g. for lat_long anomalies.

<img width="233" alt="image" src="https://user-images.githubusercontent.com/7405507/186396850-b82ac8ac-87f7-4e69-8cf0-ba53733f98fc.png">

Reply:
Good call - just keeping one of the maps links per anomalies sounds reasonable - updated in 194d15241a2663884443d87ef1f6505302ba654d